### ISSUE TITLE
Translations for other languages

### ISSUE THREAD

**Issue by pklion** [files-community/files-community.github.io#49]
Translations for other languages are not enough.
Is there someone who can make another language translation?

**Comment by chingucoding** [comment]
Is this regarding the start page (index.html)? If so, how do we want to handle all the different languages? Ideally we would have a HTACCESS file, but GitHub pages doesn't support this :/

**Comment by pklion** [comment]
I think auto redirection isn't good for all people, so establishing a dropdown menu would be better like [this](https://stopcovid19.metro.tokyo.lg.jp/en) (Tokyo Metropolitan Government COVID-19 The Information website).

**Comment by chingucoding** [comment]
I would argue that for a website with no "preferred" target audience (like the site you shared that is targeted towards people living in Tokyo) automatic detection is better, but we don't have a reasonable way to do this right now anyway. Only thing is that a dropdown might interfere with the clean design we have but there aren't many options here. If there is a dropdown, I would be more than happy to provide the German translation.

**Comment by pklion** [comment]
I found how to redirect to a translated page with javascript, so I'll make a PR.

**Comment by yaichenbaum** [comment]
@pklion Is this issue resolved?

**Comment by pklion** [comment]
@yaichenbaum No, the goal of this issue is to make other languages translation of this website.
